Sprint-2/implement/tally.test.js

```diff
@@ -32,3 +45,7 @@ test.todo("tally on an empty array returns an empty object");
 // Given an invalid input like a string
 // When passed to tally
 // Then it should throw an error
+test("return error, as the input is not an array", () => {
+  expect(tally(4353636)).toBe("Error! Input must be an array");
+  expect(tally("Hello ")).toBe("Error! Input must be an array");
+});
```

Comment:
Can you also test
`expect(tally({ a: 1, b: 2 })).toBe("Error! Input must be an array");`

Reply:
Thanks CJ for your comment. I amended my code